etc/docker-scripts/bootstrapper.py

```diff
@@ -456,6 +458,36 @@ def execute(self) -> None:
             raise ex
 
 
+class RFileOp(FileOpBase):
+    """Perform R File Operation"""
+
+    def execute(self) -> None:
```

Comment:
Not a big deal, but it seems like we could introduce a `ScriptFileOp(FileOpBase)` that defines this `execute()` method and a couple of properties (e.g., `script_lang`, `script_command`, etc.) where the properties are just substitutes in the execute method, then both PythonScriptFileOp and RScriptFileOp instances are literally just property values.

I think the properties would be generic forms of:
```
        python_script = os.path.basename(self.filepath)
        python_script_name = python_script.replace('.py', '')
 ```
We'd probably also need `script_lang` ({Python, R}) and `script_suffix` ({.py, .r})